tools/performance/benchmark-analysis/README.md

```diff
@@ -0,0 +1,5 @@
+# Analysing benchmarks
+
+This workflow is prepared mostly to analyse the output of benchmarks
+from `test/Exploratory_Benchmarks`. See `test/Exploratory_Benchmarks/README.md`
+for more information.
```

Comment:
I assume this works with the output of any of our current text based benchmarks?

Reply:
As long as their name the benchmarks using the pattern `<group>.<single-bench-name>`, then yes.

The idea of the workflow is that we display plots comparing `<single-bench-name>` inside of a single `<group>`. If the benchmarks are also grouped like that and we want to compare them within groups, then yes. If not - we'd need to adapt it but we can re-use at least the data loading part (which is just a few first nodes).